.evergreen/config.yml

```diff
@@ -974,18 +974,26 @@ functions:
           ${PREPARE_SHELL}
 
           cd ${DRIVERS_TOOLS}/.evergreen/csfle
-          source activate_venv.sh || :
+
+          # only run this if virtualenv is installed
+          if ${PYTHON3_BINARY} -m venv ./venv; then
+            . ./activate_venv.sh
+
+            # TODO (GODRIVER-2239): Stabilize this pip install with a non-forked version of PyKMIP in
+            pip install git+https://github.com/kevinAlbs/PyKMIP.git@expand_tls12_ciphers
+          else
+            echo "error: No module named venv"
```

Comment:
Optional: Is this actually an error case, or is it expected? If it's expected, consider changing the wording of this line to prevent confusion.

E.g.
> Python module venv not found, skipping virtual environment setup...